protos/cerbos/cloud/store/v1/store.proto

```diff
@@ -70,6 +71,12 @@ message GetFilesResponse {
   repeated File files = 2;
 }
 
+message ChangeDetails {
+  string origin = 1;
+  string description = 2;
+  map<string, google.protobuf.Value> metadata = 3;
```

Comment:
Would it make sense to also have a special case for changes sourced from Git rather than stuffing commit details etc into the unstructured metadata?

Reply:
I didn't want to be too prescriptive because this field is mainly for the benefit of the users and they can choose to structure the information however they like. That being said, adding a git field probably won't hurt because they can always ignore it if they don't want to use it.